lib/auth.js

```diff
@@ -169,6 +175,21 @@ bedrock.events.on('bedrock-identity.insert', function(options, callback) {
   callback();
 });
 
+
+bedrock.events.on('bedrock-identity.postInsert', function(options, callback) {
+  // TODO: Remove this and just listen for postInsert event to emit mail
+  // this hinges on postInsert being compatable with the bedrock-mail
+  // module (Identity.created uses a deprecated way of emitting events,
+  // bedrock-mail hasn't been updated to use the new way)
+  var details = {identity: options.identity};
```

Comment:
Need to store private event data in `bedrock-identity.insert` handler to check for here -- and don't emit event if not relevant.